data/devices/asus-rog-pugio.device

```diff
@@ -9,4 +9,4 @@ Buttons=10
 Leds=3
 Dpis=2
 DpiRange=100:7200@100
-ButtonMapping=f0;f1;f2;e4;e5;e6;e8;e9;e1;e2
+ButtonMapping=f0;f1;f2;e4;e5;e6;;e8;e9;e1;e2
```

Comment:
Maybe this should be a `0` explicitly?

Because otherwise we rely on `strtoul` in libratbag-data.c:376 converting an empty string to `0`, which is not completely obvious.
Example:
```c
#include <stdio.h>
#include <stdlib.h>

int main() {
  printf("%lu", strtoul("", NULL, 16));
  // > `0` printed.
}
```

Reply:
Yes. I'm fine with those changes.